Update topnav-social.html

wittigue opened the pull request:
deleted Twitter/X
added Bluesky

wittigue committed:
Update topnav-social.html

deleted Twitter/X
added Bluesky

PhilReedData commented:
Thanks @wittigue. I've checked out this branch and deployed it locally, and it still shows Twitter for me. 
<img width="578" alt="image" src="https://github.com/user-attachments/assets/72b98437-7336-4dfa-89c6-d376e483d70f" />

I've looked at my code, it is definitely showing your changes in _includes/topnav-social.html , but it is generating in _site/index.html code that still shows Twitter. 

What happens when you run it locally?

PhilReedData commented:
Also, I raised an issue in ETT GH to see if anyone was adding Bluesky as a 'native' option.
https://github.com/ELIXIR-Belgium/elixir-toolkit-theme/issues/321

I've done it manually on our eScience Lab site, I was thinking about making a PR to the ETT parent theme, when I have time. https://esciencelab.org.uk/about/

wittigue commented:
I don't have a local version and could not test it.
Thanks for checking the ETT.

PhilReedData commented:
I'm making a pull request in ETT for Bluesky now. It only needs three small changes. https://github.com/PhilReedData/elixir-toolkit-theme/releases/tag/bluesky-002 
<img width="1483" alt="image" src="https://github.com/user-attachments/assets/03572894-26ce-40bd-b5a3-f702e7cf8310" />
<img width="269" alt="image" src="https://github.com/user-attachments/assets/b538e7e2-8757-4dd1-93fb-e955d28bf52c" />
I will take out the Twitter logo for FAIRDOM. These images are for me to show Bert that my code supports both, with an updated name for those still using Twitter as X (if he wants it).

As soon as these changes are picked up in an ETT release, I'll update FAIRDOM to use that. Can we wait until then?

PhilReedData committed:
Use bluesky on development ETT fork

Will need to change remote theme to the ETT version that supports Bluesky after my PR is accepted.

PhilReedData commented:
To watch: my PR on parent ETT: https://github.com/ELIXIR-Belgium/elixir-toolkit-theme/pull/325
